l10n_nl_bsn/readme/ROADMAP.rst

```diff
@@ -0,0 +1,3 @@
+* The 'ilike' search of a substring of a BSN number could not work properly in case the
+  entered BSN number does not contain dots while the corresponding BSN number stored in the
+  database contains dots.
```

Comment:
Isn't this rather a changelog entry? So should go to `newsfragments`?

Reply:
This was meant to be a warning, explaining that the search would be not working properly in some particular cases.
In case of better solutions will be proposed, this warning could be removed one day.

Reply:
isn't this the problem you fix here?

Reply:
Yes, quick fix but not fully working: https://github.com/OCA/l10n-netherlands/pull/232#issuecomment-495118680

Reply:
ah, I missed the *substring* part. Clear now.